docusaurus/docs/shared/prerequisites.md

```diff
@@ -0,0 +1,6 @@
+- Knowledge about how data sources are implemented in the frontend.
```

Comment:
I'm not sure we should add this, these prerequisites are shared amongst data sources and panels.